package snslib;

message AppidList {
    repeated int32 appid = 1;
}

 message FlagObj {
    optional uint32 uin = 1;        // 号码
    optional int32 appid = 2;       // uin
    optional int32 ret = 3;         // 返回值，查询:返回1表示对应标志位已经设置，否则返回0，
                                    // 设置: 返回1表示设置成功，返回0表示失败
}
    
message GetFlag {
    repeated FlagObj flag_obj = 1;
}

message SetFlag {
    repeated FlagObj flag_obj = 1;
} 

message DelFlag {
    repeated FlagObj flag_obj = 1;
}